docs/releases/5.0.txt

```diff
@@ -114,6 +114,9 @@ Minor features
 :mod:`django.contrib.admin`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+* The ``admin.E109`` error now check not only many-to-many field,
+  it now also check reverse foreign key field.
```

Comment:
The grammar here is incorrect; additionally the order of the sentence should be restructured to make it flow better:

```suggestion
* The ``admin.E109`` error now checks reverse foreign key attributes in
  addition to many-to-many fields.
```

Also corrected the term field to be attribute as it's not a model field - although nessita and felixx will clarify and adjust as necessary.

Reply:
We don't add bugfixes and changes in system checks to release notes for a feature release. Please remove it.

Reply:
done